tests/DevApps/AzureFunctions/AzureFunctions.csproj

```diff
@@ -0,0 +1,30 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFramework>net8.0</TargetFramework>
```

Comment:
needs to be `TargetFrameworks>net8.0</TargetFrameworks>` with an `s` 😅otherwise it will pull in .NET 6 & 7, which we don't want.

Reply:
after you change that, there should be a warnign in VS to reload the project